src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsController.java

```diff
@@ -62,15 +59,18 @@
     public String getAddOrRemoveDirectors(@PathVariable String companyNumber,
                                           @PathVariable String transactionId,
                                           @PathVariable String companyAccountsId,
-                                          Model model,
-                                          HttpServletRequest request) {
+                                          Model model) {
 
         addBackPageAttributeToModel(model, companyNumber, transactionId, companyAccountsId);
 
+        AddOrRemoveDirectors addOrRemoveDirectors = new AddOrRemoveDirectors();
```

Comment:
Is there a reason we changed this from being autowired in the class ?

Reply:
Yes because it was being used in different places concurrently with the potential risk of the same instance being used again. Was failing some tests, once i put it in the actual methods it started passing.

Reply:
Yeah wouldn't change code for failing tests.

Reply:
Agreed with not changing code to pass tests, though autowiring this bean on the class was wrong initially; Spring would autowire a singleton by default, and we could risk concurrent requests from different clients contaminating one another, so we create a new instance per GET.